Create review-allow-entries.md

MSFTBen opened the pull request:
@MSFTTracyP - to be reviewed, pushed into "Diagnose" in nav menu under SBS.

MSFTBen committed:
Create review-allow-entries.md

@MSFTTracyP - to be reviewed, pushed into "Diagnose" in nav menu under SBS.

opbld31 commented:
Learn Build status updates of commit _[9900317](https://github.com/MSFTBen/microsoft-365-docs/commits/990031794ceb9a0d1919a46b57efc47106a08754)_: 

### :bulb: Validation status: suggestions


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :bulb:Suggestion | | [Details](#user-content-7ed5fbcc2f2fe7a1ef955fb65e672730-c7e5d93e)

<a id="7ed5fbcc2f2fe7a1ef955fb65e672730-c7e5d93e"></a>
### [microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md)
 - **Line 87, Column 36**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/microsoft-365/security/defender/advanced-hunting-overview' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 89, Column 34**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/microsoft-365/security/office-365-security/email-authentication-about' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301041410189977-10594/BuildReport?accessString=80dd39c46be4db55be0880cfe4fb92bd293483b0ce0772255482c6ce06f1ba75).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

chrisda committed:
Update review-allow-entries.md

chrisda committed:
Merge branch 'public' into patch-21

opbld30 commented:
Learn Build status updates of commit _[036084b](https://github.com/MSFTBen/microsoft-365-docs/commits/036084b931a4a076b32f0943b917d368935c8fb5)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :warning:Warning | | [Details](#user-content-7ed5fbcc2f2fe7a1ef955fb65e672730-af7ac439)

<a id="7ed5fbcc2f2fe7a1ef955fb65e672730-af7ac439"></a>
### [microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md)
 - **Line 87, Column 36**: **[Warning: file-not-found - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: 'microsoft-365/security/defender/advanced-hunting-overview'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301041844480865-10594/BuildReport?accessString=5c0cec1446b3b4d6de796e8a1c2bab4c727121b8b8a6ca6a203825e61d9298d1).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld32 commented:
Learn Build status updates of commit _[7448cbd](https://github.com/MSFTBen/microsoft-365-docs/commits/7448cbd4083ff0f0e50717bfc578f12389df3f0b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :warning:Warning | | [Details](#user-content-7ed5fbcc2f2fe7a1ef955fb65e672730-9711613b)

<a id="7ed5fbcc2f2fe7a1ef955fb65e672730-9711613b"></a>
### [microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md)
 - **Line 87, Column 36**: **[Warning: file-not-found - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: 'microsoft-365/security/defender/advanced-hunting-overview'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301041847351636-10594/BuildReport?accessString=bbd139b72ffeba2cf6e2bfe4164f79470be1569119315c9cb253af2fdc879de6).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

chrisda committed:
Update review-allow-entries.md

opbld33 commented:
Learn Build status updates of commit _[9c275b9](https://github.com/MSFTBen/microsoft-365-docs/commits/9c275b934ea16d56ee3bbba30cbc273b45477d6f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301041855314412-10594/BuildReport?accessString=5264a9a721ed2a328d6582ee2eea850e78af627a55bae1c7ee6331c98e09247b).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

Ratulch committed:
Merge branch 'public' into patch-21

opbld33 commented:
Learn Build status updates of commit _[aa14b66](https://github.com/MSFTBen/microsoft-365-docs/commits/aa14b6602e2a6364b5db6f312181db41d180e5f9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301042337051154-10594/BuildReport?accessString=ff42cee24f3b58c0895b2994ee20882aba672e6b42549ca95e3489787a04681b).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

Ratulch committed:
Merge branch 'public' into patch-21

opbld30 commented:
Learn Build status updates of commit _[2e48fbb](https://github.com/MSFTBen/microsoft-365-docs/commits/2e48fbb7cdeeb65f72d801f8b1cc4264bd5db70a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301051634000068-10594/BuildReport?accessString=b70bbe82dfdb2fa841588b97226d50272b82985f6b56772579410ca7c07c955c).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

Ratulch committed:
Merge branch 'public' into patch-21

opbld31 commented:
Learn Build status updates of commit _[0538422](https://github.com/MSFTBen/microsoft-365-docs/commits/0538422705a26907b211e70510a00d4c764e3417)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301051940101375-10594/BuildReport?accessString=9794ae624b302b4a3d0d260283fc20d5f82b248bbb049618966e0e0a02baff8f).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

MSFTTracyP committed:
Update review-allow-entries.md

Hi PubOps. I'm going to try this to see if it goes through. I'll stand by if there are any issues that need to be fixed. Thanks!

MSFTTracyP committed:
Merge branch 'public' into patch-21

opbld30 commented:
Learn Build status updates of commit _[28e205c](https://github.com/MSFTBen/microsoft-365-docs/commits/28e205c7c9ea545ff8086036328d2549ad5cec0a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301101935592950-10594/BuildReport?accessString=8ae712b4bc74320787d7f23f3e21eb573e12be4aed3616921d8665ce2447c588).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld32 commented:
Learn Build status updates of commit _[0d128eb](https://github.com/MSFTBen/microsoft-365-docs/commits/0d128ebb1f6eee6a7aca4c208ef557be9a7c4eb4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md](https://github.com/MSFTBen/microsoft-365-docs/blob/patch-21/microsoft-365/security/office-365-security/step-by-step-guides/review-allow-entries.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202301101936535399-10594/BuildReport?accessString=9a7eb8f3db67b9260526d474a93fa7a9c906a72327a22113db8780fcab4739eb).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/76d60323-5511-5963-adf9-59162c800b4a?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

MSFTTracyP committed:
Merge branch 'public' into patch-21
